vhd-format.opam

```diff
@@ -2,16 +2,16 @@ opam-version: "2.0"
 maintainer: "dave@recoil.org"
 authors: ["Dave Scott" "Jon Ludlam"]
 tags: ["org:mirage" "org:xapi-project"]
-license: "LGPL-2.1only WITH OCaml-LGPL-linking-exception"
+license: "LGPL-2.1-only WITH OCaml-LGPL-linking-exception"
 homepage: "https://github.com/mirage/ocaml-vhd"
 doc: "https://mirage.github.io/ocaml-vhd/"
 bug-reports: "https://github.com/mirage/ocaml-vhd/issues"
 depends: [
   "ocaml" {>= "4.03.0"}
-  "cstruct" {>= "1.9"}
+  "cstruct" {>= "1.9" & < "6.0.0"}
```

Comment:
I think 6.0.0 still had `len`, but 6.1.0 removed it. so an upper bound should be `< "6.1.0"`.